pkg/cloudprovider/cloudprovider.go

```diff
@@ -45,6 +47,12 @@ const (
 	linkedAuthorizationFailed retry.RetriableError = "LinkedAuthorizationFailed"
 	// Occurs when the user-assigned identity does not exist.
 	failedIdentityOperation retry.RetriableError = "FailedIdentityOperation"
+	// retryAfterHeaderKey is the retry-after header key in ARM responses.
+	retryAfterHeaderKey = "Retry-After"
+)
+
+var (
+	now = time.Now
```

Comment:
Why are we re-defining the `time.Now` function as a separate variable? Let's standardize how we grab the current time in the package by using one of `now()` and `time.Now()`.